org.eclipse.nebula.widgets.nattable.examples.e4/META-INF/MANIFEST.MF

```diff
@@ -3,8 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: NatTable Eclipse 4 Examples
 Bundle-SymbolicName: org.eclipse.nebula.widgets.nattable.examples.e4;singleton:=true
 Bundle-Version: 2.4.0.qualifier
-Require-Bundle: javax.inject;bundle-version="0.0.0",
```

Comment:
is javax.inject now part of one of the other bundles and the import resolves because of that?

Reply:
Package imports will find some appropriate bundle that exports the package without need for a bundle require.   There are jakarta bundles that export this package.

Reply:
Ah, I somehow had both Require-Bundle and Import-Package. IIRC there was an issue in the past with the one or the other. If the Import-Package statement is sufficient now, I am happy to get rid of another Require-Bundle. 

Remember that I was building against a pretty old Eclipse version and needed to solve issues with ugly workarounds. If they are not needed anymore, I am more than happy.